app/src/main/java/com/github/geohunt/app/data/repository/bounties/TeamsRepository.kt

```diff
@@ -55,6 +57,14 @@ class TeamsRepository(
 
     override suspend fun getUserTeam(): Flow<Team?> = getUserTeam(userRepository.getCurrentUser().id)
 
+    override suspend fun getUserTeamAsync(): Team = withContext(ioDispatcher) {
```

Comment:
This function is useless and wrong (the return type should be nullable). One can do `getUserTeam().first()`.